deploy/scripts/src/mobile/frontend-e2e.test.ts

```diff
@@ -38,6 +38,20 @@ const profiles: ProfileList = {
       ],
       exec: 'mamIphonePassport'
     }
+  },
+  load: {
+    mamIphonePassport: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1700, // Calculation: 100 journeys / second * 17 seconds average journey time
```

Comment:
This strikes me as quite high for the initial ramp up period (and that it's over 50% of what we'll need at peak load) - @Tom-Dann does this look ok to you?